### PR TITLE
Home page

### LINKED ISSUE #16: Create the Home page component.
**Is your feature request related to a problem? Please describe.**
N/A

**Describe how the solution should behave**
The design should look like the scrrenshoot and be responsive to all 3 different sizes. 

**Screenshots of the design/if any**
image

**Additional context**
Use andy library nneded for the carousel

### PR DESCRIPTION
**Pull request by ZhraaBaharAluloom**

### CODE REVIEW

**Comment by MuhannedNoman** [comment]
> 
> 
> _No description provided._

Can you use PRs template here so that we can get information faster!

File: src/pages/HomeSlides.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { Carousel } from "3d-react-carousal";
+import "./style.css";
+const HomeSlides = () => {
```
Comment (MuhannedNoman) [inline src/pages/HomeSlides.jsx]: Where will the book data come from? Can we have a TODO item here to remind us?

File: src/pages/HomeSlides.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  let slides = [
+    <img
```
Comment (MuhannedNoman) [inline src/pages/HomeSlides.jsx]: Can we use [next/image](https://nextjs.org/docs/api-reference/next/image) here?

**Review by MuhannedNoman — CHANGES_REQUESTED** [review]
@ZhraaBaharAluloom Hey nice work here, left you some comments to go through and let me know what you think!

**Comment by ZhraaBaharAluloom** [comment]
> > _No description provided._
> 
> Can you use PRs template here so that we can get information faster!

Sure, sorry forgot that

File: src/pages/HomeSlides.jsx (hunk 2 above)
Comment (ZhraaBaharAluloom) [reply src/pages/HomeSlides.jsx]: > Can we use [next/image](https://nextjs.org/docs/api-reference/next/image) here?

No

File: src/components/style.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
+.react-3d-carousel .slider-container .slider-right div {
```
Comment (MuhannedNoman) [inline src/components/style.css]: Just move this file next to the HomeSlider where it is used.

File: src/components/HomeSlides.jsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Carousel } from "3d-react-carousal";
+import "./style.css";
+const HomeSlides = () => {
+  // TODO: get the books images as a prop from the parent (Home)
```
Comment (MuhannedNoman) [inline src/components/HomeSlides.jsx]: Why not set it now? Have an array of objects be passed from the parent and loop through it here.

**Review by MuhannedNoman — APPROVED** [review]
LGTM, just implement the suggested changes and feel free to merge.

**Pull request merged**